admin-dev/themes/new-theme/js/components/multistore-header.ts

```diff
@@ -79,6 +80,45 @@ const initMultistoreHeader = () => {
   }
 };
 
+const initContextualNotification = () => {
+  const configKeyShopPrefix = 'missing-color-shop-';
+  const configKeyGroupPrefix = 'missing-color-group-';
+  const multistoreHeader = document.querySelector('.header-multishop');
```

Comment:
would it be better to use `.js-header-multishop`?
 or, as there is only expected to be 1 elemento, should we use id `#header-multishop`?

Reply:
Thanks. Fixed.